fastseq/config.py

```diff
@@ -24,3 +24,7 @@
 # supported versions of fairseq
 MIN_FAIRSEQ_VERSION = '0.9.0'
 MAX_FAIRSEQ_VERSION = '0.9.0'
+
+#Set following variable to use Efficient-Lossless Attention 
+USE_EL_ATTN = os.getenv('USE_EL_ATTN', '0')
```

Comment:
How about adding one more conversion here: '0' -> `False`, '1' -> `True` here? So that other places can directly use it as a boolean value.

Reply:
done